### PR TITLE
add proposed fees message

### PR DESCRIPTION
**Pull request by 0xAlcibiades**
This service will return the fee structure for using Valorem Clear and Trade. So that users can calculate relayer and other fees more easily when forming orders.

### CODE REVIEW

File: proto/valorem/trade/v1/fees.proto [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+  int32 maker_fee_rebate_notional = 2;
+  // A taker fee or rebate on premia or credit value traded expressed in basis points.
+  int32 taker_fee_rebate_premia_credit = 3;
+  // A maker fee or rebate on premia or credit value traded expressed in basis points.
+  int32 maker_fee_rebate_premia_credit = 4;
+  // A taker fee or rebate on spot value traded expressed in basis points.
+  int32 taker_fee_rebate_spot = 5;
+  // A maker fee or rebate on spot value traded expressed in basis points.
+  int32 maker_fee_rebate_spot = 6;
+  // A flat taker relayer fee or rebate expressed in USDC dust - used for non-valued offers/considerations
```
Comment (neodaoist) [inline proto/valorem/trade/v1/fees.proto]: Could clarify this is equal to 1e-6 USDC, or $0.000001.

**Review by neodaoist — APPROVED** [review]
API LGTM 👍

**Review by nickadamson — APPROVED** [review]

File: proto/valorem/trade/v1/fees.proto [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+
+package valorem.trade.v1;
+
+/* Service for getting Valorem Clear and Trade fee information. */
+service Fees {
+  // Returns the Valorem Fee structure.
+  rpc getFeeStructure(Empty) returns (FeeStructure);
+}
+
+message FeeStructure {
```
Comment (dbelv) [inline proto/valorem/trade/v1/fees.proto:13]: This message might be better structured out as:
```
message FeeStructure {
    Fees maker = 1;
    Fees taker = 2;
    // A fee or rebate on notional value written via Clear expressed in basis points.
    int32 clear_writer_notional_fee_rebate_bps = 3;
    // A fee or rebate on notional value exercised via Clear expressed in basis points.
    int32 clear_exercise_notional_fee_rebate_bps = 4;
    // The address fees must be paid to.
    H160 fee_address = 5;
}

message Fees {
    int32 fee_rebate_notional = 1;
    int32 fee_rebate_premia_credit  = 2;
    int32 fee_rebate_spot = 3;
    int32 flat_fee_rebate = 4;
}

```

**Review by dalechyn — APPROVED** [review]
lgtm

**Review by nickadamson — APPROVED** [review]

**Review by dbelv — APPROVED** [review]

**Pull request merged**